src/core/manager.c

```diff
@@ -3159,6 +3159,31 @@ static bool manager_get_show_status(Manager *m, StatusType type) {
         return false;
 }
 
+const char *manager_get_confirm_spawn(Manager *m) {
+        static bool warned = false;
+        const char *vc = m->confirm_spawn;
+        _cleanup_close_ int fd = -1;
+
+        if (!vc || path_equal(vc, "/dev/console"))
+                return vc;
+
+        /* TTYs are devices that can come and go any time, and
+         * frequently aren't available yet during early boot (consider
+         * a USB rs232 dongle...). If for any reason the configured
+         * console is not ready, we fallback to the default
+         * console. */
+        fd = open_terminal(vc, O_RDWR|O_NOCTTY|O_CLOEXEC);
+        if (fd < 0) {
+                if (!warned) {
+                        warned = true;
+                        log_warning_errno(fd, "Failed to open %s: %m, using default console", vc);
+                }
+                return "/dev/console";
+        }
+        warned = false;
+        return vc;
```

Comment:
If a serial device is removed while we are writing to it, that's tough luck. However, if the device is not around early on, then that's pretty normal and we shouldn't completely fail because of this for all future. Please add the fallback path to each time we are about to print the question.